backend/posts/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime, timedelta
+import pytz
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+from django.db.utils import IntegrityError
+
+from ..models import Vote, Post
+from ..bc import compute_post_hash, compute_vote_hash
+
+
+class TestVoteModel(TestCase):
+
+    def setUp(self):
+        User.objects.create_user('admin')
+        now = datetime.now(pytz.UTC) 
+        post_date = (now - timedelta(1)).isoformat()
+        post_title = 'Post title'
+        post_content = 'Post content'
+        post_hash = compute_post_hash('anonymous', post_date, post_title, post_content)
+        Post.objects.create(
+            author=None,
+            title=post_title,
+            content=post_content,
+            creation_datetime=post_date,
+            data_hash=post_hash,
+        )
```

Comment:
JFYK - there is another method to prepare initial records in db, by field called `fixtures`. Class needs to inherit from TransactionTestCase. Quite interesting method but anyway I recommend your way - is more explicit.

Reply:
I actually made it less explicit in the next PR as I grew tired of reiterating similar setups, so maybe I should actually consider fixtures.

Reply:
Well, now I see that I actually use `fixtures` in the next PR and I totally forgot about them. Maybe I'll use them for tests as well (not sure yet how to combine them with `pytest`)